packages/app/package.json

```diff
@@ -8,10 +8,11 @@
     "clean": "rimraf dist && rimraf ./node_modules/.vite && echo 'cleaned'",
     "clean-deps": "rimraf node_modules",
     "test": "echo 'ok'",
-    "cypress:launch": "cross-env TZ=America/New_York gulp open --project ${PWD}",
-    "cypress:open": "cross-env TZ=America/New_York gulp open --project ${PWD}",
-    "cypress:run:ct": "cross-env TZ=America/New_York node ../../scripts/cypress run --component --project ${PWD}",
-    "cypress:run:e2e": "cross-env TZ=America/New_York node ../../scripts/cypress run --project ${PWD}",
+    "cypress:run-cypress-in-cypress": "cross-env HTTP_PROXY_TARGET_HOST=http://localhost:4455 CYPRESS_REMOTE_DEBUGGING_PORT=6666 TZ=America/New_York",
+    "cypress:launch": "yarn cypress:run-cypress-in-cypress gulp open --project ${PWD}",
+    "cypress:open": "yarn cypress:run-cypress-in-cypress gulp open --project ${PWD}",
+    "cypress:run:ct": "yarn cypress:run-cypress-in-cypress node ../../scripts/cypress run --component --project ${PWD}",
+    "cypress:run:e2e": "yarn cypress:run-cypress-in-cypress node ../../scripts/cypress run --project ${PWD}",
```

Comment:
My only concern was that if we set `HTTP_PROXY_TARGET_HOST` / `CYPRESS_REMOTE_DEBUGGING_PORT` here, we might miss it if Cypress is started from the root of the project, etc.

Can we set these as additional "internal" config values that we can set on the `packages/app/cypress.config.ts`? It looks like they're only accessed when the browser is opened, so that should be fine since the config is sourced prior to that. @brian-mann thoughts?

Reply:
I would prefer not to since they're not like... official flags, and may end up getting removed anyway

Reply:
@brian-mann CYPRESS_REMOTE_DEBUGGING_PORT is the port that is used to launch the chrome debugger. I need a way to ensure that the parent and child are using the same port.

Reply:
We have a lot of these already: https://github.com/cypress-io/cypress/blob/29f5c6453cb4a0b6fdb41b861f0e86be53403a34/cli/types/cypress.d.ts#L2896-L2923 - what alternative would you suggest?

Reply:
@tgriesser Setting these on packages/app/cypress.config.ts doesn't seem to be early enough for parent cypress.